stable/velero/README.md

```diff
@@ -89,6 +89,7 @@ Parameter | Description | Default
 `metrics.serviceMonitor.enabled` | Set this to `true` to create ServiceMonitor for Prometheus operator | `false`
 `metrics.serviceMonitor.additionalLabels` | Additional labels that can be used so ServiceMonitor will be discovered by Prometheus | `{}`
 `schedules` | A dict of schedules | `{}`
+`configmaps` | Configure Velero plugins with ConfigMaps | `[]`
```

Comment:
Use `configMaps` to match the property name

Reply:
Please, change the description to Velero ConfigMaps. It is commonly used for plugins but that isn't the unique purpose.

Reply:
In my first draft version it was called ````pluginsConfigMaps```. Perhaps using that name is more clear?

Reply:
let's keep it generic `configMaps` as you initially did, analogously to other charts.

Reply:
Done